app/helpers/sessions_helper.rb

```diff
@@ -0,0 +1,20 @@
+# frozen_string_literal: true
+
+module SessionsHelper
+  def log_in(user)
+    session[:user_id] = user.id
+  end
+
+  def current_user
+    @current_user ||= User.find_by(id: session[:user_id]) if session[:user_id]
+  end
+
+  def logged_in?
+    !current_user.nil?
+  end
```

Comment:
I think that we can get rid of this method and just call `if current_user` in places where we need this.